components/credit-card/DebitCardWordProblem.tsx

```diff
@@ -1,29 +1,55 @@
 import React from 'react'
 
+import { 
+    makeArray,
+    bankArray,
+    moneyArray,
+    deductedArray,
+    bankArray2,
+    limitedArray,
+    overdraftArray,
+    protectionArray,
+    pinArray,
+    atmArray,
+    checkingArray,
+    bankArray3,
+ } from './DebitCardWordProblemData'
+import { DropDownMenu } from './DropDownMenu'
+
 const DebitCardWordProblem = () => {
   return (
     <div className="flex flex-col p-12">
-        <div className="flex justify-center mb-12">
-            <h1 className="text-green-500 font-bold text-4xl uppercase">Debit Cards</h1>
-        </div>
-        <div className="flex justify-center mb-12">
-            <h3>You charge a purchase to your debit card</h3>
-        </div>
-        <div className="grid grid-cols-3 gap-4 mb-12">
-            <span>The bank send your money to the store</span>
-            <span>insert image here</span>
-            <span>
-                Money is deducted from your checking account
-            </span>
+        <div className="mb-12">
+            <div className="flex justify-center mb-12">
+                <h1 className="text-green-500 font-bold text-5xl md:text-7xl uppercase">Debit Cards</h1>
+            </div>
+            <div className="flex justify-center mb-12">
+                <span className="text-4xl">You <DropDownMenu key={makeArray[0]} value="make" words={makeArray}/> a purchase to your debit card</span>
+            </div>
+            <div className="flex flex-col justify-center items-center">
+                <div className="px-12">
+                    <img className="" src="../../images/credit-card/debit-piggybank.svg"></img>
```

Comment:
better practice to use absolute paths instead of relative paths. in case we move this file around. 

`images/credit-card/debit-piggybank.svg` should work

Reply:
Adjusted to use absolute paths